src/Lavary/Menu/Builder.php

```diff
@@ -758,6 +758,24 @@ public function conf($key)
         return $this->conf[$key];
     }
 
+     /**
+     * Set/merge custom configuration data
+     *
+     * @param array $config
+     * @param bool $merge
+     * @param string $configGroup
+     *
+     * @return string
+     */
+    public function config(array $config, bool $merge = true, string $configGroup = 'default')
+    {
+        if($merge){
+            $this->conf = array_replace_recursive(config('laravel-menu.settings.'.$configGroup, []), $config);
```

Comment:
I think this is a little confusing. At first it looks like a recursive call to $this->config(...) where the signature doesn't match. But, instead this is a call to laravel's config(...) method. What is it doing (can you add a comment to the code?) is it saving this config globally? What's the point, does it allow easy reuse in another method?